build/charts/antrea/crds/packetsampling.yaml

```diff
@@ -0,0 +1,173 @@
+                      properties:
+                        protocol:
+                          type: integer
+                          minimum: 0
+                          maximum: 255
+                    ipv6Header:
+                      type: object
+                      properties:
+                        nextHeader:
+                          type: integer
+                          minimum: 0
+                          maximum: 65535
+                    transportHeader:
+                      type: object
+                      properties:
+                        udp:
+                          type: object
+                          properties:
+                            srcPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                            dstPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                        tcp:
+                          type: object
+                          properties:
+                            srcPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                            dstPort:
+                              type: integer
+                              minimum: 1
+                              maximum: 65535
+                            flags:
+                              type: integer
+                              minimum: 0
+                              maximum: 255
+                timeout:
+                  type: integer
+                  minimum: 1
+                  maximum: 300
+                type:
+                  type: string
+                firstNSamplingConfig:
+                  type: object
+                  properties:
+                    number:
+                      type: integer
+                fileServer:
+                  type: object
+                  properties:
+                    url:
+                      type: string
+                authentication:
+                  type: object
+                  properties:
+                    authType:
+                      type: string
+                      enum: [ "BearerToken", "APIKey", "BasicAuthentication" ]
+                    authSecret:
+                      type: object
+                      properties:
+                        name:
+                          type: string
+                        namespace:
+                          type: string
+            status:
+              type: object
+              properties:
+                reason:
+                  type: string
+                phase:
+                  type: string
+                startTime:
+                  type: string
+                numCapturedPackets:
+                  type: integer
+                packetsPath:
```

Comment:
I tried this feature locally, and found that this packetsPath is only a file name, is this expected? If yes, should it be a packetsFileName? Please double check.

Reply:
updated.